lib/actions/deploy.coffee

```diff
@@ -31,11 +31,21 @@ parseInput = Promise.method (params, options) ->
 
 	return [appName, options.build, source, image]
 
+# Builds and returns a Docker-like progress bar like this:
+# [==================================>               ] 64%
+renderProgress = (percentage, stepCount = 50) ->
+	_ = require('lodash')
+	percentage = Math.max(0, Math.min(percentage, 100))
+	barCount = stepCount * percentage // 100
+	spaceCount = stepCount - barCount
+	bar = "[#{_.repeat('=', barCount)}>#{_.repeat(' ', spaceCount)}]"
+	return "#{bar} #{percentage.toFixed(1)}%"
```

Comment:
Ok, that's fine then. I've filed an issue to fix that against the resin-cli-visuals: https://github.com/resin-io-modules/resin-cli-visuals/issues/42.